pkgs/development/libraries/givaro/default.nix

```diff
@@ -28,12 +32,16 @@ stdenv.mkDerivation rec {
     "--disable-fma"
     "--disable-fma4"
   ];
-  doCheck = true;
+
+  # needed for darwin
+  doInstallCheck = true;
+  installCheckTarget = "check";
```

Comment:
What do these do out of interest that makes it work?

Reply:
The tests are linked such that they look for the dylib file in the nix store, in the place where it will be installed. These make it so the tests get run *after* `installPhase`, so the dylib is actually there, otherwise it segfaults because it can't find it. I'm not sure why this only happens on darwin.

Reply:
Please add that explanation to the comment instead of "needed for darwin". Also explicitly set `doCheck = false` to make the intention clear and be safe if the default changes.

Reply:
I added some more detailed explanation about this to the manual in https://github.com/NixOS/nixpkgs/pull/44248